scala3doc/resources/dotty_res/scripts/bundle.js

```diff
@@ -0,0 +1,2 @@
data-key="'+t+'" data-selected="'+i.selected+'" data-value="'+r+'" data-test-id="filterGroupButton">'+r+"</button>"})).join(" ")+"\n        </div>\n      </div>\n    "},e.prototype.render=function(t){var e,n,r=this,i=t.filter;e=this.filterLowerContainerRef,n=Object.entries(i.filters).filter((function(t){t[0];var e=t[1];return Object.values(e).some((function(t){return t.visible}))})).map((function(t){var e=t[0],n=t[1];return r.getFilterGroup(e,n)})),e&&(e.innerHTML=function(t){return Array.isArray(t)?t.join(""):t}(n)),this.attachFiltersClicks(),this.attachSelectingButtonsClicks()},e}(n),C=function(){var t=function(e,n){return(t=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(t,e){t.__proto__=e}||function(t,e){for(var n in e)Object.prototype.hasOwnProperty.call(e,n)&&(t[n]=e[n])})(e,n)};return function(e,n){function r(){this.constructor=e}t(e,n),e.prototype=null===n?Object.create(n):(r.prototype=n.prototype,new r)}}(),w=function(t){function e(e){var n=t.call(this,e)||this;return n.onInputChange=function(t){var e=t.currentTarget;n.props.onInputChange(e.value)},r(".filterableInput").addEventListener("input",(function(t){return n.onInputChange(t)})),n}return C(e,t),e}(n),O=function(){var t=function(e,n){return(t=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(t,e){t.__proto__=e}||function(t,e){for(var n in e)Object.prototype.hasOwnProperty.call(e,n)&&(t[n]=e[n])})(e,n)};return function(e,n){function r(){this.constructor=e}t(e,n),e.prototype=null===n?Object.create(n):(r.prototype=n.prototype,new r)}}(),F=function(t){function e(e){var n=t.call(this,e)||this;return n.onInputChange=function(t){n.setState((function(e){return{filter:e.filter.onInputValueChange(t)}}))},n.onGroupSelectChange=function(t,e){n.setState((function(n){return{filter:n.filter.onGroupSelectionChange(t,e)}}))},n.onFilterVisibilityChange=function(){n.setState((function(t){return{isVisible:!t.isVisible}}))},n.onFilterToggle=function(t,e){n.setState((function(n){return{filter:n.filter.onFilterToggle(t,e)}}))},n.refs={elements:i(".documentableElement"),filterBar:r(".documentableFilter")},n.state={filter:new p("",{},n.refs.elements,!0),isVisible:!1},n.input=new w({onInputChange:n.onInputChange}),n.list=new y({filter:n.state.filter}),n.filterGroup=new m({filter:n.state.filter,onFilterToggle:n.onFilterToggle,onGroupSelectChange:n.onGroupSelectChange,onFilterVisibilityChange:n.onFilterVisibilityChange}),n.render(),n}return O(e,t),e.tryAttaching=function(){r(".documentableFilter")&&new e},e.prototype.render=function(){this.refs.filterBar&&(this.state.isVisible?this.refs.filterBar.classList.add("active"):this.refs.filterBar.classList.remove("active")),this.list.render({filter:this.state.filter}),this.filterGroup.render({filter:this.state.filter,onFilterToggle:this.onFilterToggle,onGroupSelectChange:this.onGroupSelectChange,onFilterVisibilityChange:this.onFilterVisibilityChange})},e}(n);window.addEventListener("DOMContentLoaded",(function(){F.tryAttaching()}))})();
```

Comment:
Can we avoid committing generated files in the repo? It's fine if the build process involves calling npm, it just means the sbt project needs to call npm itself when needed, this is what we do in the vscode-dotty project: https://github.com/lampepfl/dotty/blob/93f24a09ef22e8c150a14531dbb49e7599a70d2f/project/Build.scala#L1231-L1233

Reply:
done!